fish-rust/src/history.rs

```diff
@@ -0,0 +1,8 @@
+use crate::env::{EnvMode, EnvStack};
+use crate::wchar::prelude::*;
+
+/// Sets private mode on. Once in private mode, it cannot be turned off.
+pub fn start_private_mode(vars: &EnvStack) {
```

Comment:
should copy the docstring from the header file too

Reply:
Will fix

Reply:
Wait: which docstring do you mean here? This already has the doc from the header-file for `start_private_mode`:

https://github.com/fish-shell/fish-shell/blob/0f19d7118b10ee4354317888963c7291334e0173/src/history.h#L345

Reply:
yeah looks like it was added in a later commit (603c5b76e (Remove C++ version of `start_private_mode`, 2023-08-18))

Reply:
Ah, my bad, will move that change to the correct commit